### ISSUE TITLE
MD038 conflict with prettier

### ISSUE THREAD

**Issue by Mister-Hope** [DavidAnson/markdownlint#277]
Prettier V2 prefer:

```md
`` Ctrl + ` ``
```

while your plugin prefer:

```md
``Ctrl + ` ``
```

Each of you have a reason for doing these. (So I can not tell who is better)But I really hope you two do not conflict with each other.

Can you follow prettier's lint behavior or convince prettier to change in to your way? Thanks.

BTW, this plugin is really nice.

**Comment by DavidAnson** [comment]
Thanks! I think prettier is right. This scenario should already be supported by the library, but the spaces inside the string seem to be leading it to report a violation. I will look into this.

Example of this pattern being allowed without spaces inside the stream: https://dlaa.me/markdownlint/#%25m%23%20Issue%20277%0A%0A%60%60%20Ctrl%2B%60%20%60%60%0A
